game/screen_paths.cc

```diff
@@ -101,11 +101,10 @@ void ScreenPaths::generateMenuFromPath(fs::path path) {
 				std::clog << "screen_paths/notice: Ignoring hidden folder: ." << p.c_str() << std::endl;
 				continue;
 			} else {
-				m_menu.add(MenuOption(p.c_str(),_("Open folder")).call([this, p](){
-					generateMenuFromPath(p);
+				m_menu.add(MenuOption(boost::lexical_cast<std::string>(p),_("Open folder")).call([this, p](){ generateMenuFromPath(p);
```

Comment:
Instead of using `boost::lexical_cast<std::string>(p)` which is a value that isn't translated. use `_(p)` i think this would work since `_()`-macro returns a string and other calls to `m_menu.add(MenuOption())` are done with two translateable strings.